src/main/java/wolox/training/models/Book.java

```diff
@@ -51,6 +53,10 @@
     @Column(nullable = false, unique =true)
     private String isbn;
 
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
```

Comment:
A Book can have much users. This attribute should be List<User>

Reply:
I became it to List